app/views.py

```diff
@@ -152,8 +135,8 @@ def add_a_list():
 									})
 				return response, 201
 			return jsonify({'status': 'fail', 'message': 'list is missing in the data'}), 400
-		return jsonify({"error":"user not in isinstance"}), 401
-	return jsonify({"error":" cant access to login"}), 401
+		return jsonify({'status': 'fail', 'message':'user not in isinstance'}), 401
+	return jsonify({'status': 'fail', 'message':'cant access to login'}), 402

```

Comment:
402 status code is reserved for "Payment Required"..not sure if it fits the context of your app..ensure all your status codes make sense in the context of use and the message corresponds to the status code you're using

Reply:
changed it